spec/fixtures/user_with_disabled_either_storage.rb

```diff
@@ -0,0 +1,22 @@
+class UserWithDisabledEitherStorage < ActiveRecord::Base
```

Comment:
Although the fixtures provide some usage examples (which is good), we may want to consider bringing `factory-girl` into the mix, at some point :)